src/Umbraco.Web.UI.Client/src/common/directives/components/content/umbcontentnodeinfo.directive.js

```diff
@@ -95,23 +96,43 @@
 
                         // get current backoffice user and format dates
                         userService.getCurrentUser().then(function (currentUser) {
-                            angular.forEach(data.items, function(item) {
+                            angular.forEach(data.items, function (item) {
                                 item.timestampFormatted = dateHelper.getLocalDate(item.timestamp, currentUser.locale, 'LLL');
                             });
                         });
-                    
+
                         scope.auditTrail = data.items;
                         scope.auditTrailOptions.pageNumber = data.pageNumber;
                         scope.auditTrailOptions.pageSize = data.pageSize;
                         scope.auditTrailOptions.totalItems = data.totalItems;
                         scope.auditTrailOptions.totalPages = data.totalPages;
 
                         setAuditTrailLogTypeColor(scope.auditTrail);
-                        
+
                         scope.loadingAuditTrail = false;
                     });
 
             }
+            function loadRedirectUrls() {
+                scope.loadingRedirectUrls = true;
+                //check if Redirect Url Management is enabled
+                redirectUrlsResource.getEnableState().then(function (response) {
+                    scope.urlTrackerDisabled = response.enabled !== true;
+                    scope.userIsAdmin = response.userIsAdmin;
```

Comment:
Hi @marcemarc 

Where is this used ? Can't seem to find any uses

Dave

Reply:
It's in the view:

 <div ng-show="!hasRedirects && !loadingRedirectUrls && !urlTrackerDisabled"><localize key="redirectUrls_noRedirects"></localize></div>
                    <div ng-show="urlTrackerDisabled">Redirect Url Management is disabled</div>

Essentially I'm thinking if the redirect url management dashboard is disabled, then we shouldn't show the redirect urls here?
interestingly the getEnableState endpoint returns whether the user is an admin, but I'm doing anything with it here, it just got me thinking from your comments about not letting just anyone remove redirects, anyway as we've removed that too, shall I remove the userIsAdmin setting too for now?